packages/button-react/src/Button.test.tsx

```diff
@@ -14,41 +12,41 @@ describe("Button", () => {
     ].map((buttonVariant) => {
         it(`renders the ${buttonVariant.name} button correctly`, () => {
             const { name, component: Button } = buttonVariant;
-            const { getByText } = render(<Button onClick={() => {}}>{name}</Button>);
+            render(<Button onClick={() => {}}>{name}</Button>);
 
-            expect(getByText(name)).toHaveClass(`jkl-button--${name}`);
+            expect(screen.getByText(name)).toHaveClass(`jkl-button--${name}`);
         });
     });
 
     it("calls the onClick handler when clicked", () => {
         const clickHandler = jest.fn();
-        const { getByText } = render(<PrimaryButton onClick={clickHandler}>I am groot!</PrimaryButton>);
+        render(<PrimaryButton onClick={clickHandler}>I am groot!</PrimaryButton>);
 
-        const button = getByText("I am groot!");
+        const button = screen.getByText("I am groot!");
 
         button.click();
```

Comment:
Tror ikke dette er helt rett heller..

```
userEvent.click(screen.getByText("I am groot!"));
```

Reply:
Kan endre til `fireEvent`, (så slipper vi enda en dependency). Tipper det holder for vår bruk, i hvert fall inntil videre 😊